app/assets/stylesheets/user-styles.css

```diff
@@ -28,10 +28,10 @@ img.right {
   margin: -8px; padding: 8px;
 }
 
-#sidebar .sidebar-title {
+#sidebar h3{
```

Comment:
You are applying a new css style to a whole slew of list elements *that weren't broken in the first place*. What exaclty _was_ broken, and why?